src/content/getting-started/cloud-provider-accounts/vmware-vsphere/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+linkTitle: Preparing VMware vSphere for CAPI
+title: Preparing VMware vSphere for Cluster API
+description: How to set up your VMware vSphere Cluster tenant to run Giant Swarm management clusters and workload clusters under your jurisdiction.
+weight: 80
+user_questions:
+  - What are the VMware vSphere prerequisites for use with Giant Swarm?
+  - What do I need to configure in VMware vSphere in order to run Giant Swarm clusters?
+  - What user permissions do I need to run Giant Swarm clusters in VMware vSphere?
+menu:
+  main:
+    identifier: gettingstarted-infraprovider-vsphere
+    parent: gettingstarted-infraprovider
+last_review_date: 2023-10-03
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+---
+In order to run the Giant Swarm platform in your VMware vSphere environment, a number of prerequisites must be satisfied to support Cluster API Provider vSphere (CAPV).
+
+## vSphere infrastructure
+
+It is recommended to create resource pool for your cluster(s). However, it is possible to deploy them in the root resource pool (a.k.a. vSphere cluster).
```

Comment:
Is it clear to customers what the "root resource pool" is and why it's also called "vSphere cluster"? I find this confusing since it also contains the word "cluster" and can be mistaken for a Kubernetes cluster. Maybe remove the part in parentheses if not really needed?

```suggestion
It is recommended to create a resource pool for your cluster(s). However, it is possible to deploy them in the root resource pool (a.k.a. vSphere cluster).
```

Reply:
https://github.com/giantswarm/docs/pull/1927/commits/32b8dd1c77ee0fc48d9275e3a7cb04b188133d14